src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -215,6 +222,22 @@ public RegressionChecks generateTestChecks(ExecutableSequence eseq) {
     return checks;
   }
 
+  /**
+   * Returns true if the class (representing an array type) is a literal.
+   *
+   * @param cls -- the class to be tested
+   * @return true iff the class is a primitive, boxed primitive, String, Class, or Enum
+   */
+  private boolean isLiteralType(Class<?> cls) {
```

Comment:
Don't define this twice.  Eliminate it here or in ObserverEqArray.java.
Also, the definitions differ because in ObserverEqArray.java, no test for accessibility is performed.